### PR TITLE
Add Mindjet Mindmanager 10.3.635

### PR DESCRIPTION
**Pull request by AMammenT**
Adding current version of mindmanager.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
[Previous comment](https://github.com/caskroom/homebrew-cask/pull/10817#issuecomment-95883669) still applies.

**Comment by AMammenT** [comment]
Yikes.... Thanks.  I didn't see the earlier comments when I closed and repushed - I took out the comments in the update.  Will fix.

File: Casks/mindjet-mindmanager.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'mindjet-mindmanager' do
+  version :latest # 10.3.365 as of Apr 2015
```
Comment (vitorgalvao) [inline Casks/mindjet-mindmanager.rb]: The comment should be removed. It’ll be quickly outdated.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by AMammenT** [comment]
Sqaushed and removed the comment.

**Pull request merged**